demo/screens/EthereumSignUp.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import {
+  Image,
+  Button,
+  StyleSheet,
+  View,
+  Text,
+  TextInput,
+  KeyboardAvoidingView,
+} from "react-native";
+import {
+  createStackNavigator,
+  createAppContainer,
+  Header,
+} from "react-navigation";
+
+export default class EthereumSignUp extends React.Component {
+  state = {
+    text: "",
+  };
+
+  render() {
+    return (
+      <KeyboardAvoidingView
+        keyboardVerticalOffset={Header.HEIGHT + 20}
+        style={styles.container}
+        behavior="padding"
+      >
+        <Text style={styles.text}>
+          {`Cool. Let's start by picking your Tasit username.`}
+        </Text>
+        <View style={styles.userRow}>
+          <View style={styles.userInputView}>
+            <TextInput
```

Comment:
Once arg I like is disabling capitalization and autocorrection for inputs like this

Reply:
Sure!